tutorials/katacoda/thanos/3-receiver/step2.md

```diff
@@ -0,0 +1,98 @@
+# Setup Thanos Receive
+
+With `prometheus-batcave` & `prometheus-batcomputer` now running, we need to think about how we satisfy our two requirements:
+1. Implement a global view of this data.
+1. Global view must be queryable in near real-time.
+
+How are we going to do this?
+
+## Thanos Sidecar
+
+After completing [Tutorial #1: Global View](https://www.katacoda.com/thanos/courses/thanos/1-globalview), you may think of running the following architecture:
+
+* Run Thanos Sidecar next to each of the Prometheus instances.
+* Configure Thanos Sidecar to upload Prometheus data to an object storage (S3, GCS, Minio).
+* Run Thanos Store connected to the data stored in object storage.
+* Run Thanos Query to pull data from Thanos Store.
+
+However! This setup **does not** satisfy our requirements above.
+
+<details>
+ <summary>Can you think why?</summary>
+
+Thanos Sidecar only uploads `blocks` of metrics data that have been written to disk, which happens every 2 hours in Prometheus.
```

Comment:
This gives the implication that you can never get near real-time data using sidecars, which might confuse people who are new to Thanos. Maybe we can add something like `Since we cannot connect to sidecars from outside the clusters, ....` to make it more explicit why sidecars don't solve our problem in this particular problem.

Reply:
Right, this is the impression I got from the tutorial so excellent point.

Reply:
Yeah good call out - will update now :+1: